spec/requests/api/v1/course_spec.rb

```diff
@@ -398,5 +410,31 @@ def get_next_courses(link, params = {})
         end
       end
     end
+
+    context 'with new courses' do
+      let(:current_cycle) { RecruitmentCycle.current_recruitment_cycle }
+      let(:provider1) { create(:provider, recruitment_cycle: current_cycle) }
+      let(:provider2) { create(:provider, recruitment_cycle: current_cycle) }
+
+      let(:course1) { create(:course, study_mode: 'full_time', profpost_flag: 'postgraduate', program_type: 'higher_education_programme', provider: provider1) }
+      let(:course2) { create(:course, study_mode: 'full_time', profpost_flag: 'postgraduate', program_type: 'higher_education_programme', provider: provider1) }
+
+      let(:status1) { create(:site_status, status: :new_status, course: course1, vac_status: 'full_time_vacancies', applications_accepted_from: Date.new(2019, 7, 23)) }
+      let(:status2) { create(:site_status, status: :new_status, course: course2, vac_status: 'full_time_vacancies', applications_accepted_from: Date.new(2019, 7, 24)) }
+      let(:status3) { create(:site_status, status: :running, course: course2, vac_status: 'full_time_vacancies', applications_accepted_from: Date.new(2019, 7, 24)) }
+
+      it 'does not send courses marked new' do
+        status1
+        status2
+        status3
```

Comment:
use `let!` instead of `let` to cause immediate execution of the block, this will eliminate the need for calling them at the start of the `it` block

Reply:
or a before block, its confusing looking at

```ruby

it 'test something' do
do_lots_of_setup

do_actual_thing

expect(actual).to eq (do_lots_of_setup)

```